.gitlab-ci.yml

```diff
@@ -50,14 +50,47 @@ e2e:
   tags:
     - kube
   script:
+    - cp $E2E_COMPOSE_FILE ./docker-compose.yaml
+    - cp $E2E_CONFIG_FILE ./e2e.yaml
```

Comment:
you'll have to test to verify, but since those are CI variables, you may instead want:

`echo $VARNAME > ./file.yaml`

Reply:
When you set the variable type as `file` it creates a temp file with the content and sets the env var with that name to the path of the temp file. It is working as is now.